libsolidity/lsp/FileRepository.cpp

```diff
@@ -51,6 +67,8 @@ string FileRepository::sourceUnitNameToUri(string const& _sourceUnitName) const
 		return "file:///" + _sourceUnitName;
 	else if (_sourceUnitName.find("/") == 0)
 		return "file://" + _sourceUnitName;
+	else if (auto const resolvedPath = tryResolvePath(_sourceUnitName); resolvedPath.message().empty())
```

Comment:
Isn't this supposed to kick in when the resolved path is **not** empty?
```suggestion
	else if (auto const resolvedPath = tryResolvePath(_sourceUnitName); !resolvedPath.message().empty())
```